utils/update_checkout/update-checkout-config.json

```
@@ -177,7 +180,8 @@
                 "curl": "curl-8_9_1",
                 "libxml2": "v2.11.5",
                 "zlib": "v1.3.1",
-                "mimalloc": "v3.0.3"
+                "mimalloc": "v3.0.3",
+                "swift-subprocess": "main"
```

@compnerd @shahmishal I think we want this assuming this is the right syntax for depending on a tag called "development-snapshot-2025-07-21"?
```suggestion
                "swift-subprocess": "development-snapshot-2025-07-21"
```